doc/bloom/tutorial.adoc

```diff
@@ -202,3 +202,37 @@ For AND combination, be aware that the resulting FPR will be in general
 worse (higher) than if the filter had been constructed from scratch
 by inserting only the commom elements -- don't trust `fpr_for` in this
 case.
+
+== Direct Access to the Array
+
+The contents of the bit array can be accessed directly with the `array`
+member function, which can be leveraged for filter serialization:
+
+[listing,subs="+macros,+quotes"]
```

Comment:
Unless I'm recalling incorrectly, we should only need this when we're creating code blocks with cross-references, which I don't think we're using here.

This might avoid the need for the `pass:[]` macro invocation.